pyhf/optimize/__init__.py

```diff
@@ -49,6 +49,8 @@ def __getattr__(self, name):
                     "There was a problem importing Minuit. The minuit optimizer cannot be used.",
                     e,
                 )
+        elif name == '__wrapped__':  # doctest
```

Comment:
@kratsg Can you add a simple/short explanation of where `__wrapped__` pops up in doctest to the PR description or the commit summary?

Reply:
https://bugs.python.org/issue25998 and https://github.com/timofurrer/ramlient/issues/51 explain some of the issues well here. The comment in the code + the context of the comment in the code is rather explanatory. I've had to add an exception for `__wrapped__` because doctesting is checking attributes on module as part of doctesting.

Reply:
https://bugs.python.org/issue25998 is helpful, thanks. Given the context of these two resources I now agree with you that the comment is enough. Without them though it wasn't clear what was happening that this was actually happening.